.circleci/config.yml

```diff
@@ -685,3 +717,174 @@ workflows:
             - test-jruby-9.2.0.0
             - test-jruby-9.2-latest
             # soon™️ - test-truffleruby-21.0.0
+  edge:
+    triggers:
+      - schedule:
+          cron: "0 0 * * 1-5" # Every weekday
+          filters:
+            branches:
+              only:
+                - master
+    jobs:
```

Comment:
Would it be possible to add an `&ci_jobs` id to the jobs above, and here just use `jobs: *ci_jobs`?

There's already quite a lot of copy/paste going on to maintain this list, would be great if we could avoid yet another thing that needs to be kept in sync.

Reply:
Unfortunately the value expected in the `jobs:` field is an array, and there's no way to merge arrays in YAML. In other words, we can't have `ci_jobs` with a list of jobs but also add a few more to it (`coverage`, `lint`, `deploy release`) 😢

Reply:
😭

Reply:
Might be worth documenting this -- e.g. in both sections, state "This is exactly the same thing as [below/above], but with the [foo/bar/baz] sections [added/removed]", so it's harder to forget to update them, and to confuse why they are different.